AppCenter/AppCenter/Internals/MSServiceCommon.h

```diff
@@ -124,6 +102,28 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)isStartedFromApplication;
 
```

Comment:
good catch! `isStartedFromApplication` is not optional.

Reply:
I got surprised that we did this one.